Components/MineSharp.Protocol/Packets/Clientbound/Play/SetExperiencePacket.cs

```diff
@@ -0,0 +1,37 @@
+using MineSharp.Core.Serialization;
+using MineSharp.Data;
+using MineSharp.Data.Protocol;
+
+namespace MineSharp.Protocol.Packets.Clientbound.Play;
+
+/// <summary>
+///     Set Experience packet
+/// </summary>
+/// <param name="ExperienceBar">The experience bar value between 0 and 1</param>
+/// <param name="Level">The experience level</param>
+/// <param name="TotalExperience">The total experience points</param>
+public sealed record SetExperiencePacket(float ExperienceBar, int Level, int TotalExperience) : IPacket
```

Comment:
for some reason, the order in which ExperienceBar and Level are sent changed in 1.19.2 and then again in 1.20.2